domador.js

```diff
@@ -659,4 +659,12 @@ Domador.prototype.isVisible = function isVisible (el) {
   return visible;
 };
 
+Domador.prototype.openCodeFence = function openCodeFence (el) {
+  var fencinglanguage = this.options.fencinglanguage(el);
+  if (el.childNodes.length > 0 && el.childNodes[0].tagName === 'CODE') {
```

Comment:
wait, don't you mean `var child = el.childNodes[0]; if (el && el.childNodes && el.childNodes[0] && el.childNodes[0].tagName === 'CODE')` ? Not that that's elegant at all, but wouldn't `el.childNodes[0]` still throw an error if the el has no childNodes?

Thanks for the tips!